client/src/components/profile/info.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import {
+  Container, Button, Row, Col, Spinner,
+} from 'react-bootstrap';
+import Avatar from 'react-avatar';
+import PropTypes from 'prop-types';
+import '../../style/profile-info.css';
+
+class Info extends Component {
+  render() {
+    const { isFetching, user } = this.props;
+    if (isFetching) {
+      return (
+        <Container fluid>
+          <Row className="profile-row">
+            <Col md="6">
+              <p>
+                Name:
+                {' '}
+                { user.name }
+              </p>
+              <p>
+                Email:
+                {' '}
+                { user.email }
+              </p>
+              <p>
+                Phone:
+                {' '}
+                { user.phoneNumber }
+              </p>
```

Comment:
I think it is better create 'UserInfo' component and reuse lines 17-31 there.

Reply:
It is userinfo component (info.js) and you can reuse it everywhere, you only need to send object with properties `email, name, etc` into this component.
Splitting this component more makes code and project more complicated and doesn't bring any advantages
And now this code works and does their function - show information about User

Then and only then when we need to split this component we will do it https://martinfowler.com/bliki/Yagni.html

Reply:
Sorry, I was mistaked - we can use this component `as UserInfo` component anywhere. 

Anyway, lines 18, 23 and 28 are hardcoded. It is better:
1. to define them in default prop values;
2. use `.map()` method to retrieve all incoming prop values.

Reply:
Sorry, I don't understand you. I did my task and why I need to think about "something anywhere"? Right now this component is used only in one place. And I can't see future (unfortunately) and I don't now if we will need to use this component in other place of project.

About creating menu lines from array. There are only 3 menu items (avatar has other structure). Creating additional abstraction layer brings more complexity than advantages in this case. If we will see (in the future) that we need more menu items we can change it in any time

Reply:
Ok